### DISCUSSION TITLE
Not possible to save new QSO - "...is not focusable"

### DISCUSSION THREAD

**Discussion by LarsThunberg** [magicbug/Cloudlog#1108] [General]
Hello,
I have just done a new installation of Cloudlog without any problems. When I'm trying to save a new QSO I get several errors in the console of Chrome, and the QSO can not be saved. The debug information is all green and PHP version is 7.4.21.
Any help are welcome. See screenshot below.
73 Lars SM0TGU

![image](https://user-images.githubusercontent.com/35871385/127118848-598485bd-f9e9-409e-b40d-0b77f988c9a0.png)

**Comment by magicbug** [comment]
Just woke up so brains still hazy, few things have you created a station location and also updated country files via Admin - Update Country Files?

**Reply by LarsThunberg** [reply]
Thank you, it fixed the problem. Missed that one during the installation, sorry...
73 Lars
